docs/use-cases.rst

```diff
@@ -196,6 +196,93 @@ Two parameters to be configured in this type of implementation are outlined:
 |    :end-before: <!--><-->                              |    :end-before: <!--><-->                              |
 +--------------------------------------------------------+--------------------------------------------------------+
 
+.. _discovery_server_redundancy_scenario_setup:
+
+UDPv4 redundancy example
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`above example <discovery_server_major_scenario_setup>` presents a *single point of failure*, that is, if the
+*server* falls there is no discovery. In order to prevent this, several servers could be linked to a *client*. By doing
```

Comment:
```suggestion
*server* fails there is no discovery. In order to prevent this, several servers could be linked to a *client*. By doing
```

Reply:
Fall and fail imply different things in this context but, given that I speak of failure later on, let's buy it.